### DISCUSSION TITLE
Install test dependencies from txt file

### DISCUSSION THREAD

**Discussion by tvdboom** [pdm-project/pdm#3210] [Q&A] [answered]
Hi,

I have a very long list of dependencies in a `requirements.txt` file that I want pdm to install as part of a dev dependency group. I don't see anywhere in the docs how to achieve this. I'm looking for something like in the image. Any help is appreciated.

![image](https://github.com/user-attachments/assets/04f1683d-d722-4a7e-a681-9ebaf7f18770)

where `requirements.txt` looks like this (with over 300 dependecies, that's why I don't want it in `pyproject.toml`).

![image](https://github.com/user-attachments/assets/cd2f79bf-fd44-4555-897a-86b1d34a95ee)

**Comment by frostming** [comment, marked as answer]
Just run `pdm import` to import it into pyproject.toml beforehand. It supports `--dev` and `--group` option to specify which group to import into.
